spec/system/product_drive_participant_system_spec.rb

```diff
@@ -25,19 +25,24 @@
       expect(page.find(:xpath, "//table/tbody/tr[3]/td[1]")).to have_content(@third.business_name)
     end
 
-    context "When the s have donations associated with them already" do
+    it "allows single participants to show comments" do
+      visit product_drive_participant_path(product_drive_participant)
+      expect(page).to have_xpath("//table/tbody/tr/td", text: @first.comment)
+    end
```

Comment:
This doesn't need to be a (heavy) system test. It can be a request test.

Reply:
resolved by [635fb81](https://github.com/rubyforgood/human-essentials/pull/4442/commits/635fb81593e111fd8541bbb263a2fb5c020c89e8)